code/game/objects/structures/headpike.dm

```diff
@@ -37,6 +36,11 @@
 		MA.pixel_y = 12
 		add_overlay(H)
 
+/obj/structure/headpike/examine(mob/user)
+	. = ..()
+	if(user?.mind.has_antag_datum(/datum/antagonist/ashwalker))
```

Comment:
Would be nice if antag datum registered to listen for these directly.

Reply:
Sadly not possible right now, unless I use arguably worse workarounds. I don't think we have a signal that triggers when a mob examines something, only when an atom gets examined.

https://github.com/tgstation/tgstation/blob/master/code/game/atoms.dm#L299
I would basically would have to add SEND_SIGNAL(user, COMSIG_MOB_EXAMINE, src) for this to work, which would be really odd and confusing.

Reply:
Done.